test/test-production.js

```diff
@@ -29,6 +29,7 @@ require('can-symbol/can-symbol-test');
 require('can-reflect/can-reflect-test');
 require('can-reflect-promise/test/can-reflect-promise_test');
 require('can-attribute-encoder');
+require('can-ajax');
```

Comment:
This should be the test file. Also.. the one above this is wrong, should import the tests for `can-attribtue-encoder` if you want to fix that up also.